lib/main.dart

```diff
@@ -1,143 +1,78 @@
 import 'package:flutter/material.dart';
-import 'package:flutter/rendering.dart';
+import 'SecondView.dart';
 
 void main() {
-  runApp(MyApp());
+  runApp(MaterialApp(
+    home: MyApp(),
+  ));
 }
 
-class MyApp extends StatelessWidget {
+class MyApp extends StatefulWidget {
   @override
-  Widget build(BuildContext context) {
-    return MaterialApp(
-      home: MainView(),
-    );
-  }
-}
-
-class MainView extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: (Colors.blue),
-      appBar: AppBar(
-        centerTitle: true,
-        iconTheme: IconThemeData(color: Colors.black),
-        backgroundColor: Colors.yellow,
-        title: Text('To-Do List',
-            style: TextStyle(color: Colors.black, 
-            fontSize: 30)
-            ),
-        actions: [
-          IconButton(
-            icon: Icon(Icons.more_vert, 
-            color: Colors.black),
-            onPressed: () {
-              showDialog(
-                context: context,
-                builder: (BuildContext context) {
-                return AlertDialog(
-                  title: Text('All  Done  Undone'),
-
-                  //Enda funktionen jag inte riktigt lyckades implementera var drop-down-menyn för att se "All", "Done" och "Undone". Tacksam för feedback på hur man bäst gör det :)
```

Comment:
Tips! Byt ut IconButton() till en PopUpMenuButton(), sen gör en itembuilder som returnar tre PopUpMenuItem() (all, done, undone). Att sedan lyckas med att få varje knapptryck att visa tre olika listor beroende på om de är done/undone, det är rätt krångligt, jag löste det genom att göra två olika listor: en normal lista med alla todo-items, sen en separat lista som består av de todo-items som ska synas i fönstret. Om du vill kan du gå in och kika på min kod på gusholroa för att se hur jag fick till det.